### PR TITLE
feat: fetch 100 items per page

### PR DESCRIPTION
**Pull request by caugner**
Also makes this configurable via MERGESTAT_PARAMS.

### CODE REVIEW

File: syncs/github-dependabot/main.ts [hunk 1]
```diff
@@ -41,12 +42,14 @@ const octokit = new OctokitWithThrottling({
...
 const alertsBuffer = [];
+const perPage = params.perPage || 100;
 
 const iterator = octokit.paginate.iterator(`GET /repos/${owner}/${repo}/dependabot/alerts`, {
     headers: {
         'X-GitHub-Api-Version': '2022-11-28'
     },
-    owner, repo
+    owner, repo,
+    per_page: perPage,
```
Comment (caugner) [inline syncs/github-dependabot/main.ts:52]: See: https://docs.github.com/en/rest/dependabot/alerts?apiVersion=2022-11-28#list-dependabot-alerts-for-a-repository--parameters

File: syncs/github-issue-events/main.ts [hunk 2]
```diff
@@ -41,12 +42,14 @@ const octokit = new OctokitWithThrottling({
...
 const eventsBuffer = [];
+const perPage = params.perPage || 100;
 
 const iterator = octokit.paginate.iterator(`GET /repos/${owner}/${repo}/issues/events`, {
     headers: {
         'X-GitHub-Api-Version': '2022-11-28'
     },
-    owner, repo
+    owner, repo,
+    per_page: perPage,
```
Comment (caugner) [inline syncs/github-issue-events/main.ts:52]: See: https://docs.github.com/en/rest/issues/events?apiVersion=2022-11-28#list-issue-events-for-a-repository--parameters

File: syncs/github-repo-info/main.ts [hunk 3]
```diff
@@ -57,8 +58,11 @@ const { data: latestRelease } = await octokit.rest.repos.getLatestRelease({
     }
 });
 
+const perPage = params.perPage || 100
+
 const releasesIter = octokit.paginate.iterator(octokit.rest.repos.listReleases, {
-    owner, repo
+    owner, repo,
+    per_page: perPage,
```
Comment (caugner) [inline syncs/github-repo-info/main.ts:65]: https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#list-releases--parameters

File: syncs/github-issue-comments/main.ts [hunk 4]
```diff
@@ -41,12 +42,14 @@ const octokit = new OctokitWithThrottling({
...
 const commentsBuffer = [];
+const perPage = params.perPage || 100
 
 const iterator = octokit.paginate.iterator(`GET /repos/${owner}/${repo}/issues/comments`, {
     headers: {
         'X-GitHub-Api-Version': '2022-11-28'
     },
-    owner, repo
+    owner, repo,
+    per_page: perPage,
```
Comment (caugner) [inline syncs/github-issue-comments/main.ts:52]: See: https://docs.github.com/en/rest/issues/comments?apiVersion=2022-11-28#list-issue-comments-for-a-repository--parameters

**Review by caugner — COMMENTED** [review]
Note that REST API requests seem to be rate limited on a per-request basis, see: https://docs.github.com/en/rest/overview/rate-limits-for-the-rest-api?apiVersion=2022-11-28#primary-rate-limit-for-authenticated-users

File: syncs/github-issues/main.ts [hunk 5]
```diff
@@ -46,7 +46,7 @@ const octokit = new OctokitWithGrapQLPagination({
 const issuesBuffer = [];
 
 const iterator = octokit.graphql.paginate.iterator(query, {
-    owner, repo, perPage: 30
+    owner, repo, perPage: 100
```
Comment (patrickdevivo) [inline syncs/github-issues/main.ts:49]: @caugner could we make this configurable thru the `MERGESTAT_PARAMS` env as well?

File: syncs/github-pull-requests/main.ts [hunk 6]
```diff
@@ -46,7 +46,7 @@ const octokit = new OctokitWithGrapQLPagination({
 const prsBuffer = [];
 
 const iterator = octokit.graphql.paginate.iterator(query, {
-    owner, repo, perPage: 30
+    owner, repo, perPage: 100
```
Comment (patrickdevivo) [inline syncs/github-pull-requests/main.ts:49]: and here as well?

File: syncs/github-issues/main.ts (hunk 5 above)
Comment (caugner) [reply syncs/github-issues/main.ts]: Fixed in 2429c3c.

File: syncs/github-pull-requests/main.ts (hunk 6 above)
Comment (caugner) [reply syncs/github-pull-requests/main.ts]: Fixed in 2429c3c.

**Review by patrickdevivo — APPROVED** [review]

**Pull request merged**